apps/test/unit/templates/curriculumCatalog/CurriculumCatalogTest.jsx

```diff
@@ -293,12 +300,28 @@ describe('CurriculumCatalog', () => {
 describe('CurriculumCatalog with url params', () => {
```

Comment:
Love that you created a new describe block for readability. I do have a question though: Thoughts about putting this inside the `CurriculumCatalog` block? So this describe block could just read `with url params` and then you wouldn't need duplicate setup

Reply:
I think it might get a little messy since for the url ones I need to set the windowlocation (which changes for each url test but is not used in the other tests) so I feel like in trying to broaden out the first setup to encompass the second it would add complexity that might get in the way of readability? But I'm open to pushback for sure!

Reply:
Ah, I hear that. I was thinking something like this (where a nested describe block also contains an afterEach() and/or beforeEach()):
https://github.com/code-dot-org/code-dot-org/blob/195996b9bf26bccf0d568b67813e2839537314c5/apps/test/unit/code-studio/pd/form_components_func/FormControllerTest.js#L390-L418

I haven't seen the pattern of having totally separate `describe` blocks within the same test file, so that's what I'm reacting to. But if it's not working, I don't think it's worth forcing it.

Reply:
Oh gotcha gotcha, on it thank you!